packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx

```diff
@@ -55,7 +55,6 @@ export const MultiTrackSidebar = () => {
                 className={styles.iconError}
                 icon={IconError}
                 size='xSmall'
-                fill='accentRed'
```

Comment:
How come this doesn't work? Wondering if we can update the icon component to make it work?

Reply:
not sure. I can take a look though and see if its easy to fix

Reply:
Awesome yeah lmk